tests/e2e/fine_grained_access/common.py

```diff
@@ -11,6 +11,14 @@
 import mgclient
 
 
+def reset_permissions(admin_cursor):
+    execute_and_fetch_all(admin_cursor, "REVOKE LABELS * FROM user;")
+    execute_and_fetch_all(admin_cursor, "REVOKE EDGE_TYPES * FROM user;")
+    execute_and_fetch_all(admin_cursor, "MATCH(n) DETACH DELETE n;")
+    execute_and_fetch_all(admin_cursor, "CREATE (n:test_delete {name: 'test1'});")
+    execute_and_fetch_all(admin_cursor, "CREATE (n:test_delete_1)-[r:edge_type_delete]->(m:test_delete_2);")
```

Comment:
It is confusing why `MATCH` and `CREATE` are used in a function called `reset_permissions`, they don't hahve to anything with permissions.